README.md

```diff
@@ -1,6 +1,16 @@
 # demo
-Banks headquartered in oil-heavy states increased loan loss provisioning well above the national median in the 2015 fourth quarter. But since reserves are coming up from a low base, some analysts think banks will need to further increase provisioning to deal with the energy fallout.
-In Texas, nearly 100,000 people are employed in oil and gas extraction, representing more than half the nationwide total. Banks in the state increased provisioning by 71.94% year-over-year to $1.17 billion in 2015. 
+Banks headquartered in oil-heavy states increased loan loss provisioning ```well above the national median```
+
+*Does this compare a median to an aggregate?*
+
```

Comment:
Yes, that's right, but we can reconcile. How about:

Banks headquartered in oil-heavy states increased loan loss provisioning at a far greater rate than U.S. banks generally in the 2015 fourth quarter.